src/Categories/Category/Instance/Properties/Setoids/Limits/Canonical.agda

```diff
@@ -0,0 +1,62 @@
+{-# OPTIONS --without-K --safe #-}
+
+-- A "canonical" presentation of limits in Setoid.
+--
+-- These limits are obviously isomorphic to those created by
+-- the Completeness proof, but are far less unweildy to work with.
+-- This isomorphism is witnessed by Categories.Diagram.Pullback.up-to-iso
+
+module Categories.Category.Instance.Properties.Setoids.Limits.Canonical where
+
+open import Level
+
+open import Data.Product using (_,_; _×_)
+
+open import Relation.Binary.Bundles using (Setoid)
+open import Function.Equality as SΠ renaming (id to ⟶-id)
+import Relation.Binary.Reasoning.Setoid as SR
+
+open import Categories.Diagram.Pullback
+
+open import Categories.Category.Instance.Setoids
+
+open Setoid renaming (_≈_ to [_][_≈_])
+
+--------------------------------------------------------------------------------
+-- Pullbacks
+
+record FiberProduct {o ℓ} {X Y Z : Setoid o ℓ} (f : X ⟶ Z) (g : Y ⟶ Z) : Set (o ⊔ ℓ) where
```

Comment:
In a subsequent PR, I think it would make sense to promote `FiberProduct` to be something less buried. I'm needing it elsewhere, and this seems like a odd place to dig it out of!